test/tests/functional/pbs_qsub_opts_args.py

```diff
@@ -54,6 +54,23 @@ def setUp(self):
         self.qsub_cmd = os.path.join(
             self.server.pbs_conf['PBS_EXEC'], 'bin', 'qsub')
 
+    def validate_error(self, err):
+        ret_msg = 'qsub: Failed to save job/resv, '\
+            'refer server logs for details'
+        # PBS returns 15161 error code when it fails to save the job in db
+        # but in PTL_CLI mode it returns modulo of the error code.
+
+        # for PTL_API mode
+        if e.rc == 15161:
+            self.assertFalse(e.msg[0], ret_msg)
+        # for PTL_CLI mode
+        elif 15161 % 256 == e.rc:
```

Comment:
Why not combine these into one if statement with an 'or' between them?  Also, swap the two sides of the '==' to be consistent with the above check.